modules/ROOT/pages/prereq-software.adoc

```diff
@@ -80,16 +81,6 @@ Ensure that no processes or files conflict with those settings.
 
 == Third Party Software Prerequisites
 
-=== Install LVM2
-
-To install and run Anypoint Platform PCE, you must install and use Logical Volume Manager 2 (LVM2). LVM2 is a tool that adds a layer of abstraction between your operating system and the disks and partitions it uses. You must have root access to install LVM2.
-
-You can install LVM2 using Yum. Yum is an open-source command-line package-management utility for Linux operating systems using the RPM Package Manager. Using Yum, you can install LVM2 using the following command:
-
-----
-sudo yum install lvm2
-----
-
 === Requirements When Firewalld Is Enabled
 
 Firewalld is an iptables controller that defines rules for persistent network traffic.
```

Comment:
In line 148 it should say 3.0 instead of 2.1 in the installer url

Reply:
Fixed.